subcommand/tls-init/commant_test.go

```diff
@@ -0,0 +1,591 @@
+package tls_init
```

Comment:
The file is named `commant_test.go`, can you rename to `command_test.go`?

Reply:
oh lol..that is a really good catch!!